tslearn/svm.py

```diff
@@ -169,8 +192,10 @@ def fit(self, X, y, sample_weight=None):
         sklearn_X = _prepare_ts_datasets_sklearn(X)
         if self.kernel == "gak" and self.gamma == "auto":
             self.gamma = gamma_soft_dtw(to_time_series_dataset(X))
-            self.kernel = _kernel_func_gak(sz=self.sz, d=self.d, gamma=self.gamma)
-        return super(TimeSeriesSVC, self).fit(sklearn_X, y, sample_weight=sample_weight)
+            self.kernel = _sparse_kernel_func_gak(sz=self.sz, d=self.d, gamma=self.gamma)
+        _self = super(TimeSeriesSVC, self).fit(sklearn_X, y, sample_weight=sample_weight)
```

Comment:
Why create a new variable here? Why not just do something like:

```python 
super(TimeSeriesSVC, self).fit(sklearn_X, y, sample_weight=sample_weight)
self.kernel = _sparse_kernel_func_gak(sz=self.sz, d=self.d, gamma=self.gamma, slice_support_vectors=self.support_)
return self
```

Reply:
You are right, no need for that extra variable :)

Reply:
OK, could you commit accordingly?

Reply:
Done!